rebuild_std.sh

```diff
@@ -18,7 +18,7 @@ cargo install --debug --path ./checker
 
 # build the mirai-standard-contracts crate
 touch standard_contracts/src/lib.rs
-RUSTC_WRAPPER=mirai RUST_BACKTRACE=1 MIRAI_LOG=warn MIRAI_START_FRESH=true MIRAI_SHARE_PERSISTENT_STORE=true cargo build --lib -p mirai-standard-contracts
+RUSTFLAGS="-Z force-overflow-checks=off" RUSTC_WRAPPER=mirai RUST_BACKTRACE=1 MIRAI_LOG=warn MIRAI_START_FRESH=true MIRAI_SHARE_PERSISTENT_STORE=true cargo build --lib -p mirai-standard-contracts
```

Comment:
Yes, I think this is required. Most programs don't care about checking for overflow (and in reality, it most of the time doesn't matter).

Reply:
In this case it is only to make it easier to use Rust to write summaries for functions that do no check for overflow.